tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_math_binary_api.h

```diff
@@ -0,0 +1,86 @@
+//     EltwiseBinaryReuseDestType binary_reuse_dest = EltwiseBinaryReuseDestType::NONE>
+// inline void llk_math_eltwise_binary_init(const std::uint32_t transpose = 0, const std::uint32_t acc_to_dest = 0) {
+//     const std::uint32_t num_faces = 4;
+
+//     _llk_math_eltwise_binary_init_<eltwise_binary_type, src_b_bcast_type, NUM_FIDELITY_PHASES, binary_reuse_dest>(
+//         num_faces, transpose, acc_to_dest);
+// }
+
+// // Version with operands
+// template <
+//     EltwiseBinaryType eltwise_binary_type,
+//     BroadcastType src_b_bcast_type,
+//     int NUM_FIDELITY_PHASES = 0,
+//     EltwiseBinaryReuseDestType binary_reuse_dest = EltwiseBinaryReuseDestType::NONE>
+// inline void llk_math_eltwise_binary_init_with_operands(
+//     const std::uint32_t operand_A,
+//     const std::uint32_t operand_B,
+//     const std::uint32_t transpose = 0,
+//     const std::uint32_t acc_to_dest = 0) {
+//     const std::uint32_t operand_id =
+//         get_operand_id(operand_A);  // operand_id is used to extract tile dim data which is the same for both operands
+//     const std::uint32_t num_faces = get_operand_num_faces(operand_id);
+
+//     _llk_math_eltwise_binary_init_<eltwise_binary_type, src_b_bcast_type, NUM_FIDELITY_PHASES, binary_reuse_dest>(
+//         num_faces, transpose, acc_to_dest);
+// }
+
+// template <
+//     EltwiseBinaryType eltwise_binary_type,
+//     BroadcastType src_b_bcast_type,
+//     DstSync Dst = DstSync::SyncFull,
+//     int NUM_FIDELITY_PHASES = 0,
+//     EltwiseBinaryReuseDestType binary_reuse_dest = EltwiseBinaryReuseDestType::NONE,
+//     bool is_fp32_dest_acc_en = false>
+// inline void llk_math_eltwise_binary(uint dst_index, const bool clear_fp32_dst_acc = true) {
+//     const std::uint32_t num_faces = 4;
+
+//     _llk_math_eltwise_binary_<
+//         eltwise_binary_type,
+//         src_b_bcast_type,
+//         Dst,
+//         NUM_FIDELITY_PHASES,
+//         binary_reuse_dest,
+//         is_fp32_dest_acc_en>(num_faces, dst_index, clear_fp32_dst_acc);
+// }
+
+// template <
+//     EltwiseBinaryType eltwise_binary_type,
+//     BroadcastType src_b_bcast_type,
+//     DstSync Dst = DstSync::SyncFull,
+//     int NUM_FIDELITY_PHASES = 0,
+//     EltwiseBinaryReuseDestType binary_reuse_dest = EltwiseBinaryReuseDestType::NONE,
+//     bool is_fp32_dest_acc_en = false>
+// inline void llk_math_eltwise_binary(
+//     const std::uint32_t operand_A,
+//     const std::uint32_t operand_B,
+//     uint dst_index,
+//     const bool clear_fp32_dst_acc = true) {
+//     const std::uint32_t operand_id = get_operand_id(operand_A);  // both operands must have same number of faces
+//     const std::uint32_t num_faces = get_operand_num_faces(operand_id);
+
+//     _llk_math_eltwise_binary_<
+//         eltwise_binary_type,
+//         src_b_bcast_type,
+//         Dst,
+//         NUM_FIDELITY_PHASES,
+//         binary_reuse_dest,
+//         is_fp32_dest_acc_en>(num_faces, dst_index, clear_fp32_dst_acc);
+// }
```

Comment:
delete?

Reply:
Hey @DrJessop, all these grayskull api files are needed for compile. The grayskull api updates are happening in another branch here: https://github.com/tenstorrent-metal/tt-metal/pull/4270/

This PR is for wormhole_b0 api uplift, so the GS api files are empty but required for compile until they are ready in the other branch